model/schema/mod.yaml

```diff
@@ -0,0 +1,70 @@
+id: https://github.com/alliance-genome/agr_curation_schema/src/schema/mod
+
+imports:
+  - core
+  - linkml:types
+  - informationContentEntity
+  - agent
+  - reference
+
+prefixes:
+  alliance: 'http://alliancegenome.org'
+  linkml: 'https://w3id.org/linkml/'
+  gff: 'https://w3id.org/gff'
+  faldo: 'http://biohackathon.org/resource/faldo#'
+  biolink: 'https://w3id.org/biolink/vocab/'
+  NLMID: 'https://www.ncbi.nlm.nih.gov/nlmcatalog/?term='
+  schema: 'http://schema.org/'
+
+classes:
+  Mod:
+    is_a: Organization
+    description: >-
+      A Model Organism Database that is a member of the Alliance of Genome Resources.
+    notes: >-
+      We have not modeled synonyms, e.g. Mouse Genome Informatics
+      Literature has 8 values: FB, MGI, RGD, SGD, WB, XB, ZFIN, GO.
+    slots:
+      - abbreviation
+      - date_created
+      - date_updated
```

Comment:
There are already date_last_modified and creation_date slots in the core.yaml.  I see from the reference.yaml that these are intentional aliases - is this just because it would be too hard at this point to rename these fields in the literature DB?

Reply:
@markquintontulloch Yeah, we've spent the first half of the release working to get things to the point where we can add a new API endpoint, and I think we're there just the day before review.  We only have the rest of the release to work on everything else that needs to happen for Literature, so we have renaming the tables to be singular instead of plural in the backlog.  Chris G requested we make the PubMed-based Biblio date fields proper date fields instead of strings, and I'm pushing for that to happen, as I've already parsed the data, but we can't even prioritize doing a fix for that (it's in the backlog).  Doing a renaming of the internal date_created / date_updated created by continuum only affects the LinkML and it probably isn't even in anyone's radar.